portal-web/test/functional/com/liferay/portalweb/macros/headless/blogs/HeadlessBlogAPI.macro

```diff
@@ -87,4 +92,67 @@ definition {
 		return "${entryId}";
 	}
 
+	macro _getAllBlogsPostingId {
```

Comment:
maybe `BlogPostingAPI.getBlogPostingsIdList()` could be helpful here. With this method `_getAllBlogsPostingId` what I see you receive is the whole response with blog postings body, not only the ID.